src/java/org/apache/cassandra/config/Config.java

```diff
@@ -302,6 +302,8 @@
     private static boolean isClientMode = false;
     private static Supplier<Config> overrideLoadConfig = null;
 
+    public Integer temporary_cache_size_in_mb;
```

Comment:
We'll have to include a short section on this in `cassandra.yaml`, correct? Also, I'm guessing we'll have to make a couple minor changes to the description of `file_cache_size_in_mb`. For example...

```
32MB of this are reserved for pooling buffers, the rest is used as an
cache that holds uncompressed sstable chunks.
```

(Also there's a typo there, should be "a" not "an")

Reply:
+1